Core/src/main/java/com/plotsquared/core/command/FlagCommand.java

```diff
@@ -576,20 +577,22 @@ public void list(
         for (final Map.Entry<String, ArrayList<String>> entry : flags.entrySet()) {
             Collections.sort(entry.getValue());
             Component category =
-                    MINI_MESSAGE.parse(
+                    MINI_MESSAGE.deserialize(
                             TranslatableCaption.of("flag.flag_list_categories").getComponent(player),
-                            Template.of("category", entry.getKey())
+                            TemplateResolver.templates(Template.template("category", entry.getKey()))
```

Comment:
Is TemplateResolver required here?

Reply:
Yes, #deserialize expects a TemplateResolver.